guides/source/action_view_overview.md

```diff
@@ -622,7 +646,7 @@ You can use the same technique to localize the rescue files in your public direc
 Since Rails doesn't restrict the symbols that you use to set I18n.locale, you can leverage this system to display different content depending on anything you like. For example, suppose you have some "expert" users that should see different pages from "normal" users. You could add the following to `app/controllers/application_controller.rb`:
```

Comment:
This I believe is no longer true, as I18n by default does validate the locales, you can't set it to "anything". The recommended way to achieve something like that would be to use `variants` nowadays. (though we probably can leave that to layouts & rendering, unless a gentle introduction here would makes sense.)

Reply:
By `variants` do you mean having views like `show.html+mobile.erb`?

I think I will remove this "anything you like" example. It does not feel relevant in this guide. The previous 2 paragraphs covey the main point that it's possible to render localized views. And then linking to the I18n guide at the end for more examples. Let me know if you feel differently, I can dig into the I18n guide and craft an example with variant.

Reply:
Yeah that's what I mean about `variants`.

I think it's okay to remove this paragraph / example. 👍